packages/home-service/src/resolver.ts

```diff
@@ -1,31 +1,44 @@
 import { Home } from './schema';
-import { pubsub } from './helpers';
-
-const Home_UPDATE = 'Home_UPDATE';
 
 export const HomeResolver = {
   Query: {
     // queries
-    list(root: any, args: any, ctx: any) {
-      return [{message: 'GET API for Home microservice'}];
+    listHomeType(root: any, args: any, ctx: any) {
+      return Home.find()
+      .then(response => response)
+      .catch(err => err);
     },
-    get(root: any, args: any, ctx: any) {
+    getHomeType(root: any, args: any, ctx: any) {
       // fetch the id from args.id
-      return {message: 'GET by ID API for Home microservice'};
+      return Home.findById(args._id)
+      .then(response => response)
+      .catch(err => err);
+    },
+    getHomeTypeBy(root: any, args: any, ctx: any) {
+      return Home.find(args.input).exec();
     }
   },
   Mutation: {
     // mutations
```

Comment:
Organize the comments well always put comment above the proposed update

Reply:
Noted and changed